### DISCUSSION TITLE
iOS平台使用QuantumultX添加Siri 服务，显示invalid line

### DISCUSSION THREAD

**Discussion by lge6** [NSRingo/.github#28] [Q&A 问答]
iOS平台使用QuantumultX添加最新的Siri重写，添加过程中显示：Invalid Line• Siri.Search.request。
![04077E6D-4E01-4306-89B2-A756662E047E_1_102_o](https://github.com/user-attachments/assets/54ad401c-b942-4d71-9043-0eae7f576b77)

**Comment by VirgilClyne** [comment]
fixed
